CHANGES_NEXT_RELEASE

```diff
@@ -3,4 +3,6 @@
 - Update Howtos with the last changes of the IoT Agent library (#48).
 - Add test cases to check first observation of a value (#54).
 - Update to master branch of IoTAgent Library and LWM2M Library 0.5.0.
-- FEATURE update node version to 4.8.4
\ No newline at end of file
+- FEATURE update node version to 4.8.4
+- Unit tests are executed as part of Travis CI.
+- npm-shrinkwrap file is updated (#99 and #100)
```

Comment:
Not sure if I'm understanging correctly... You mean that updating npm-shrinkwrap was the solution for issue #99 and #100?

Reply:
Yes, that's correct. Since the npm-shrinkwrap was not updated, actually still the old version of https://github.com/telefonicaid/iotagent-node-lib was used causing these bugs.

Reply:
- Fixing issues in the provisioning API (#99 and #100) as result of npm-shrinkwrap file update.

We typically don't mention updates of npm-shrinkwrap file in CNR, but this case I think is a good idea, given doing so is fixing issues.

(I understand that #99 and #100 are about bugs in the provisioning API, otherwise my suggestion would be wrong. Please, check)

Reply:
Fixed in 9ccd48635f74f7bb63cd6ea9bc540527e500e817